src/rules/prefer-params-over-parent-id.ts

```
@@ -167,6 +167,57 @@ export const preferParamsOverParentId = createRule<[], MessageIds>({
+      // Check if params is destructured from the event parameter
+      if (
+        handlerNode.type === AST_NODE_TYPES.ArrowFunctionExpression ||
+        handlerNode.type === AST_NODE_TYPES.FunctionExpression ||
+        handlerNode.type === AST_NODE_TYPES.FunctionDeclaration
+      ) {
+        const firstParam = handlerNode.params[0];
+        if (!firstParam) return false;
+
+        // Check for destructuring pattern: ({ params }) or ({ data, params })
+        if (firstParam.type === AST_NODE_TYPES.ObjectPattern) {
+          return firstParam.properties.some(prop => {
```

ESLint

[error] Replace `prop` with `(prop)`

(prettier/prettier)